priv/templates/phx.gen.auth/notifier.ex

```diff
@@ -1,21 +1,29 @@
 defmodule <%= inspect context.module %>.<%= inspect schema.alias %>Notifier do
-  # For simplicity, this module simply logs messages to the terminal.
-  # You should replace it by a proper email or notification tool, such as:
-  #
-  #   * Swoosh - https://hexdocs.pm/swoosh
-  #   * Bamboo - https://hexdocs.pm/bamboo
-  #
-  defp deliver(to, body) do
-    require Logger
-    Logger.debug(body)
-    {:ok, %{to: to, body: body}}
+  import Swoosh.Email
+
+  alias <%= inspect context.base_module %>.Mailer
+
+  defp deliver(email, subject, body) do
+    result =
+      new()
+      |> to(email)
+      |> from({"MyApp", "contact@example.com"})
+      |> subject(subject)
+      |> text_body(body)
+      |> Mailer.deliver()
+
+    case result do
+      :ok -> {:ok, %{to: email, body: body}}
+      {:ok, _} -> {:ok, %{to: email, body: body}}
+      {:error, _} = error -> error
+    end
```

Comment:
Why so many different results? 🤔

Reply:
Btw, we should just return the whole email. We previously returned `%{to: email, body: body}` exactly so it matched the email fields. :)

Reply:
Yeah, it was because the `Mailer.deliver/1` didn't have a clear API, so I was trying to match the standard result tuples. But I think it's fair to return just the result of the deliver :).